processor/blockprocessor/block_processor.go

```diff
@@ -195,14 +233,16 @@ func prepareEvalResources(l *indexerledger.LedgerForEvaluator, block *bookkeepin
 }
 
 // Preload asset and app creators.
-func prepareCreators(l *indexerledger.LedgerForEvaluator, payset transactions.Payset) (map[basics.AssetIndex]ledger.FoundAddress, map[basics.AppIndex]ledger.FoundAddress, error) {
+func prepareCreators(lfe *indexerledger.LedgerForEvaluator, payset transactions.Payset) (map[basics.AssetIndex]ledger.FoundAddress, map[basics.AppIndex]ledger.FoundAddress, error) {
 	assetsReq, appsReq := accounting.MakePreloadCreatorsRequest(payset)
-
-	assets, err := l.GetAssetCreator(assetsReq)
+	assets, err := lfe.GetAssetCreator(assetsReq)
+	if err != nil {
+		return nil, nil, fmt.Errorf("prepareCreators() err: %w", err)
+	}
+	apps, err := lfe.GetAppCreator(appsReq)
 	if err != nil {
 		return nil, nil, fmt.Errorf("prepareCreators() err: %w", err)
 	}
-	apps, err := l.GetAppCreator(appsReq)
 	if err != nil {
 		return nil, nil, fmt.Errorf("prepareCreators() err: %w", err)
 	}
```

Comment:
Am I just seeing a weird diff artifact, or is this checking the exact same thing twice on l:243 and then l:246?

Reply:
Thanks for noticing. I thought this was an artifact, but a quick check of the actual source confirms that this is truly duplicative!